index.html

```diff
@@ -0,0 +1,206 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css">
+    <link rel="stylesheet" href="./assets/css/style.css">
+    <link rel="stylesheet" href="./assets/css/mobile.css">
+    <link rel="stylesheet" href="./assets/css/desktop.css">
+   
+    <title>Global Summit home Page</title>
+</head>
+
+<body class="container">
+    <header class="headerContainer">
+        <nav class="navbar">
+
+            <div class="firstNavBar">
+                <ul class="navlist">
+                    <li><a href="#facebook"><img src="./assets/img/someDesktopImg/facebook.svg" alt="facebook"></a></li>
+                    <li><a href="#twitter"><img src="./assets/img/someDesktopImg/twitter.svg" alt="twiter"></a></li>
+                    <li>English</li>
+                    <li>My Page</li>
+                    <li>Logout</li>
+                </ul>
+            </div>
+
+            <button class="hamburguer" type="button"><img src="./assets/img/hamburguer.svg" alt="hamburguer icon"></button>
+            <div class="secondNavbar">
+                <div class="logoContainer">
+                    <a href="index.html"><img src="./assets/img/logo.png" alt="Summit logo"></a>
+                </div>
```

Comment:
Great job @Graciano1997 🤝

- [ ] As stated in the project requirements, kindly endeavor to stick to the design guidelines as much as possible. You may remove the logo from both the Home and the About versions of your mobile views.
- Your design
![your-mobile](https://github.com/Graciano1997/global-summit/assets/63917725/fbc9b500-41be-4745-9933-346a1ad40b0f)
- Given design
![given-mobile](https://github.com/Graciano1997/global-summit/assets/63917725/ce21a5c4-69a6-4bfc-93e8-d4a7cf6a470f)

Reply:
thanks for the recommendation